scripts/add.data.sh

```diff
@@ -21,6 +21,7 @@ addInputFile "${FQDN}" "297" "FAO_" "${FAO_INPUT}"
 addInputFile "${FQDN}" "296" "OGE2_" "${OGE2_INPUT}"
 
 # [76] EBIFARM
+addInput "76" "24" "2004-01-01" "2009-12-31"
```

Comment:
Where is this input used? If you want to reference it use ${INPUT_ID} which is the id of the newly added input